### PR TITLE
SPCTR-3408: Fix - Container background image x-position issue

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Contianer background image is not set to extreme right even if the x-position is set to max.
https://brainstormforce.atlassian.net/browse/SPCTR-3408

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by bsf-zanev — COMMENTED** [review]
A few minor comments, and changelog update.

File: README.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Fix: Conatiner - Resolved and issue wherein the background image custom x-position would not work as intended.
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply README.md]: Done.

File: classes/class-uagb-block-helper.php [hunk 2]
```diff
@@ -1572,7 +1573,7 @@ public static function uag_get_background_obj( $bg_obj, $css_for_pseudo = '' ) {
 							$position_value                    = $position['x'] * 100 . '% ' . $position['y'] * 100 . '%';
 							$gen_bg_css['background-position'] = $position_value;
 						} elseif ( 'custom' === $custom_position && isset( $x_position ) && isset( $y_position ) && isset( $x_position_type ) && isset( $y_position_type ) ) {
-							$position_value                    = $x_position . $x_position_type . ' ' . $y_position . $y_position_type;
+							$position_value                    = false === $bg_obj['centralizedPosition'] ? $x_position . $x_position_type . ' ' . $y_position . $y_position_type : 'calc( 50% +  ' . $x_position . $x_position_type . ') calc( 50% + ' . $y_position . $y_position_type . ')';
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply classes/class-uagb-block-helper.php]: Updated as per suggestion.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Improvement: Conatiner - Added a new 'Centralized Position' setting for custom image background positioning.
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply README.md]: Done.

**Pull request merged**